accounts/client.go

```diff
@@ -45,6 +65,25 @@ func (c *Client) CreateEntity(request OnboardEntityRequest) (*OnboardEntityRespo
 	return &response, nil
 }
 
+func (c *Client) RetrievePaymentInstrumentDetails(entityId string, paymentInstrumentId string) (*PaymentInstrumentDetailsResponse, error) {
```

Comment:
Relocate this method after `QueryPaymentInstruments` so that we maintain some consistency

Reply:
Also this line is too long. Refactor to:
```
func (c *Client) RetrievePaymentInstrumentDetails(
    entityId string, 
    paymentInstrumentId string,
) (*PaymentInstrumentDetailsResponse, error) {
```

Reply:
👍